.github/workflows/main.yml

```diff
@@ -28,47 +28,19 @@ jobs:
         python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
I think the django 3.2 version here can be updated to the latest maintenance version (3.2.16 as per tests)